canvas_course_info/requirements/aws.txt

```diff
@@ -1,9 +1,9 @@
-Django==1.8.15
-dj-static==0.0.6
-django-redis-cache==1.7.0
+Django==1.10.5
+django-redis-cache==1.7.1
 git+https://github.com/harvard-dce/dce_lti_py.git@v0.7.4#egg=dce-lti-py==0.7.4
 git+https://github.com/harvard-dce/django-auth-lti.git@dce-lti-py#egg=django-auth-lti==1.0
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-ui.git@v1.3.1#egg=django-icommons-ui==1.3.1
 hiredis==0.2.0
 redis==2.10.5
-requests==2.11.1
-voluptuous==0.8.7
+requests==2.13.0
+voluptuous==0.9.3
```

Comment:
That's a smart approach to keeping things up-to-date on tools that are relatively low touch.  I just didn't realize that was part of your process, but now I know :)